app/models/tipline_newsletter.rb

```diff
@@ -273,7 +273,9 @@ def validate_header_file(max_size, allowed_types, message)
 
   def not_scheduled_for_the_past
     if self.content_type == 'static' && self.scheduled_time.past?
-      errors.add(:send_on, I18n.t(:send_on_must_be_in_the_future))
+      field = :send_on
+      field = :time if self.scheduled_time.strftime('%Y-%m-%d') == Time.now.utc.strftime('%Y-%m-%d')
+      errors.add(field, I18n.t(:send_on_must_be_in_the_future))
```

Comment:
Unrelated to this PR, but just for consistency, mark the problematic field as "time" instead of "day" if a datetime is in the past but the day is the same as today.